app/src/main/java/com/example/week4_threadcoroutine/MainActivity.kt

```diff
@@ -0,0 +1,80 @@
+package com.example.week4_threadcoroutine
+
+import android.os.Bundle
+import android.os.Handler
+import android.os.Message
+import androidx.appcompat.app.AppCompatActivity
+import com.example.week4_threadcoroutine.databinding.ActivityMainBinding
+import kotlin.concurrent.thread
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+    private var started = false
+
+    // 상수 선언
+    private val SET_TIME = 51
+    private val RESET = 52
+
+    // 핸들러 선언
+    private val handler = object : Handler() {
+        override fun handleMessage(msg: Message) {
+            when (msg.what) {
+                SET_TIME -> {
+                    binding.textTimer.text = formatTime(msg.arg1)
+                }
+                RESET -> {
+                    binding.textTimer.text = "00 : 00"
+                }
+            }
+        }
+    }
```

Comment:
Handler를 사용할 때 Handler의 기본생성자-`Handler()`의 경우 deprecated가 된 함수라 경고가 뜨고 있는 것 같습니다. 핸들러가 메시지를 꺼내올 Looper를 지정해주는 식으로 선언해주세요!

```suggestion
    private val handler = object : Handler(Looper.getMainLooper()) {
        override fun handleMessage(msg: Message) {
            when (msg.what) {
                SET_TIME -> {
                    binding.textTimer.text = formatTime(msg.arg1)
                }
                RESET -> {
                    binding.textTimer.text = "00 : 00"
                }
            }
        }
    }
```